relay-server/src/utils/nel.rs

```diff
@@ -0,0 +1,87 @@
+//! Contains helper function for NEL reports.
+
+use chrono::{DateTime, Duration, Utc};
+use relay_event_schema::protocol::{
+    Contexts, Event, HeaderName, HeaderValue, Headers, LogEntry, NelContext, NetworkReportRaw,
+    Request, ResponseContext, Timestamp,
+};
+use relay_protocol::Annotated;
+
+/// Enriches the event with new values using the provided [`NetworkReportRaw`].
+pub fn enrich_nel_event(event: &mut Event, nel: Annotated<NetworkReportRaw>) {
```

Comment:
Could this function be moved into `relay-normalize`? 

Either way works, as there's currently no clear criteria for where normalization is called and where the code is located. Particularly, this is not part of the current "store normalizer" but rather called during deserialization like for security reports.

However, in an effort to reduce the amount of schema-specific business logic that resides in `relay-server`, a better place for these sorts of functions would be the normalize crate.

Reply:
Yeah, I can move it to normalize. It was initially there, but then was moved back to the server utils.